script/migrations/78-deploy_KINTO_bridged_token.s.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.18;
+
+import {LibString} from "solady/utils/LibString.sol";
+import {ERC20} from "@openzeppelin-5.0.1/contracts/token/ERC20/ERC20.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {BridgedToken} from "../../src/tokens/bridged/BridgedToken.sol";
+import {IKintoWallet} from "../../src/interfaces/IKintoWallet.sol";
+import {MigrationHelper} from "@kinto-core-script/utils/MigrationHelper.sol";
+import {UUPSProxy} from "@kinto-core-test/helpers/UUPSProxy.sol";
+import {console2} from "forge-std/console2.sol";
+
+contract DeployKintoScript is MigrationHelper {
+    using LibString for *;
+    using stdJson for string;
+
+
+    // KINTO_WALLET will be the admin, minter and upgrader of every BridgedToken
+    address kintoWallet = vm.envAddress("ADMIN_KINTO_WALLET");
+    address admin = kintoWallet;
+    address minter = admin;
+    address upgrader = admin;
+
+    function run() public override {
+        super.run();
+
+        string memory symbol = 'KINTO';
+        string memory name = 'Kinto Token';
+
+        // deploy token
+        bytes memory bytecode = abi.encodePacked(type(BridgedToken).creationCode, abi.encode(decimals));
```

Comment:
need to modify the code to add the transfer requirements

Reply:
Yep. Done.